### PR TITLE
Added a How to install Appwrite on a Linode cloud server.

### PR DESCRIPTION
**Pull request by Gregor-Davies**
## What does this PR do?

This PR added a how to install appwrite on a linode cloud server tutorial.

## Test Plan

You can follow the steps in the tutorial.

## Related PRs and Issues

This PR is connected to the appwrite #4011 issue (https://github.com/appwrite/appwrite/issues/4011)

### Have you read the [Contributing Guidelines on issues

Yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [Integrate Adalo with Appwrite](https://imknight.com/integrate-adalo-with-appwrite/)
 * [How to install and configure Appwrite on CyberPanel](https://tafadzwa.hashnode.dev/how-to-install-and-configure-appwrite-on-cyberpanel)
 * [How to install Appwrite on Amazon EC2 instance](https://awstip.com/installing-appwrite-on-amazon-ec2-instance-13f6744f2b48)
+* [How to install Appwrite on a Hetzner cloud server](https://dev.to/gregordavies/how-to-install-appwrite-on-a-hetzner-cloud-server-in-8-steps-1lan)
```
Comment (gewenyu99) [inline README.md:147]: Ooops. Looks like you have a little something left over here!

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Hi Gregor!

like the other PR, great writing! It's fun and humorous and I love the comics.

I'd like to suggest a quick grammar review, if possible, before I accept this PR. I think the writing is a little too loose for us to accept as is.

I think getting someone to proof read or using Grammarly are easy ways to clean this up a little. After that, I'd love to accept his awsome PR

Thanks a lot!!!

**Comment by Gregor-Davies** [comment]
> t writing! It's fun and humorous and I love the comics.
> 
> I'd like to suggest a quick grammar review, if possible, before I accept this PR. I think the writing is a little too loose for us to accept as is.
> 
> I think getting someone to proof read or using Grammarly are

Hello, I've tightened up my grammar with Grammarly. Hope this fixes all the mistakes :)

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
It looks like you've added two links instead of one, please remove line # 147 

* [How to install Appwrite on a Hetzner cloud server](https://dev.to/gregordavies/how-to-install-appwrite-on-a-hetzner-cloud-server-in-8-steps-1lan)

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by Gregor-Davies** [comment]
> It looks like you've added two links instead of one, please remove line # 147
> 
> * [How to install Appwrite on a Hetzner cloud server](https://dev.to/gregordavies/how-to-install-appwrite-on-a-hetzner-cloud-server-in-8-steps-1lan)

Oops my bad fixed

**Review by gewenyu99 — APPROVED** [review]
Perfect, everything's tidied up!

Thank you for contributing @Gregor-Davies !

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**